workload/ui/index.html

```diff
@@ -1,132 +1,173 @@
-          {{! --- Apps --- }}
-          {{#each apps}}
-              <li class="p-matrix__item">
-			  {{#if name}}
-              <div class="p-matrix__img">
-                <span class="iconify icon md-48" data-icon="mdi-{{icon}}"></span>
-              </div>
-              <div class="p-matrix__content">
-                <h3 class="p-matrix__title"><a class="p-matrix__link" href="{{url}}">{{name}}</a></h3>
-                {{#if description}}
-                <p class="p-matrix__desc">{{description}}</p>
-                {{/if}}  
-              </div>
-			  {{/if}}
-              </li>
-            {{/each}}
-        </ul>
-        {{else}}
-        <div class="p-notification--caution">
-          <div class="p-notification__content">
-            <h5 class="p-notification__title">No items to display</h5>
-            <p class="p-notification__message">No applications available for display yet. Add some by relating compatible charms to this one.</p>
-          </div>
+    </header>
+
+    <div class="p-strip--suru">
+      <div class="row">
+        <div class="col-12">
+          {{#if tagline}}
+            <h1>{{tagline}}</h1>
+          {{/if}}
+          {{#if description}}
+            <p>{{description}}</p>
+          {{/if}}
         </div>
-        <span>
-          
-        </span>
-        {{/if}}
       </div>
     </div>
-    {{#if links}}
-    <div class="row">
-      <div class="col12">
-        <h3>Links</h3>
+
+    <div class="p-strip">
+      <div class="row">
+        <div class="col12">
+          <h3>Applications</h3>
+        </div>
       </div>
-    </div>
-    <div class="row">
-      <div class="col-12">
-        <ul class="p-matrix" id="links">
-          {{! --- Links --- }}
-          {{#each links}}
-          <li class="p-matrix__item">
-            {{#if category}}
-            <div class="p-matrix__img">
-              <span class="iconify icon md-48" data-icon="mdi-bookmark"></span>
-            </div>
-            <div class="p-matrix__content">
-              <h3 class="p-matrix__title">{{category}}</h3>
-              <ul class="link-list">
-                {{#each items}}
-                <li>
-                  <a href="{{url}}" {{#if target}}target="{{target}}"{{/if}}>
-                    {{name}}
-                  </a>
+      <div class="row">
+        <div class="col-12">
+          {{#if apps}}
+            <ul class="p-matrix" id="apps">
+              {{#each apps}}
+                <li class="p-matrix__item">
+                  <div class="p-matrix__img">
+                    <span class="iconify icon md-48" data-icon="mdi-{{icon}}"></span>
+                  </div>
+                  <div class="p-matrix__content">
+                    <h3 class="p-matrix__title">{{name}}</h3>
+                    <div class="" style="display: flex; gap: 0;">
+                      <button class="p-tooltip--btm-center" id="ui-button" {{#unless url}}disabled{{/unless}}>
```

Comment:
Not sure if rendering a greyed-out button for all missing items is good or bad, but we can keep it like this for now. One short-term benefit is forcing discoverability :)